mpt_extension_sdk/api/builders/event.py

```
@@ -15,35 +15,34 @@
+        return create_task_event_route(route, extension_app)
+    return create_non_task_event_route(route, extension_app)
 
-    Args:
-        path: The absolute URL path for this route.
-        task_handler: The task event handler function.
-        extension_app: The extension app.
 
-    Returns:
-        A configured `APIRouter` instance.
-    """
+def create_task_event_route(  # noqa: WPS213, WPS217
```

Ruff (0.15.12)

[warning] Invalid rule code in `# noqa`: WPS213, WPS217

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)